internal/testing/testdata/testdata.go

```diff
@@ -264,6 +335,76 @@ var (
 		},
 	}
 
+	slsa1time, _ = time.Parse(time.RFC3339, "2023-01-01T12:34:56Z")
+	SlsaPreds1   = assembler.IngestPredicates{
+		IsOccurrence: []assembler.IsOccurrenceIngest{
+			{
+				Src: &model.SourceInputSpec{
+					Type:      "git",
+					Namespace: "github.com/octocat/hello-world@refs/heads",
+					Name:      "main",
+				},
+				Artifact: &model.ArtifactInputSpec{
+					Algorithm: "gitCommit",
+					Digest:    "c27d339ee6075c1f744c5d4b200f7901aad2c369",
+				},
+				IsOccurrence: &slsaIsOccurrence,
+			},
+			{
+				Pkg: &model.PkgInputSpec{
+					Type:      "guac",
+					Namespace: ptrfrom.String("generic"),
+					Name:      "_",
+					Version:   ptrfrom.String(""),
+					Subpath:   ptrfrom.String(""),
+				},
+				Artifact: &model.ArtifactInputSpec{
+					Algorithm: "sha256",
+					Digest:    "fe4fe40ac7250263c5dbe1cf3138912f3f416140aa248637a60d65fe22c47da4",
+				},
+				IsOccurrence: &slsaIsOccurrence,
+			},
+		},
+		HasSlsa: []assembler.HasSlsaIngest{
+			{
+				Artifact: &model.ArtifactInputSpec{
+					Algorithm: "sha256",
+					Digest:    "fe4fe40ac7250263c5dbe1cf3138912f3f416140aa248637a60d65fe22c47da4",
+				},
+				Builder: &model.BuilderInputSpec{
+					Uri: "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@refs/tags/v0.0.1",
+				},
+				Materials: []model.ArtifactInputSpec{{
+					Algorithm: "gitCommit",
```

Comment:
is "gitCommit" ok here? see: https://github.com/in-toto/attestation/blob/main/spec/v1.0/digest_set.md

Reply:
Should be.